include/queryfunctions.php

```diff
@@ -233,7 +258,7 @@ function StartWorkoutFromProgram($workoutId){  // takes a program and duplicates
             workouts
         WHERE 
             workoutId = ".$workoutId."
-        ");
+        ");/*
```

Comment:
Based on your comment, I take it you want to keep this commented out code, yeah?